### DISCUSSION TITLE
export to pdf or html

### DISCUSSION THREAD

**Discussion by wy1189** [elimintz/justpy#414] [Q&A]
Hi,

Is there any way to export the app into pdf or html version?

Also, I'm wondering how I can convert highcharts objects to html?

**Comment by elimintz** [comment]
I am not sure what you mean by exporting the app. Could you please elaborate?

For highcharts please see: https://www.highcharts.com/docs/export-module/export-module-overview

**Comment by wy1189** [comment]
Hi, thanks for your quick reply.

What I meant was there is any way to export the charts into pdf or html format. In other words, is there a way to download/export the web page? I'm trying to share the app without web access.

So, if I generated this example using justpy

![image](https://user-images.githubusercontent.com/9031310/142692620-71b1c05b-55e4-4a49-8d0a-9185048d4889.png)

But, I want to share this result with other people in a pdf format.

And, as for Highcharts, thank you for sending me the link, but I was asking that the highcharts objects can be exported to HTML files within justpy functions such as to_html or something like that.

**Comment by WolfgangFahl** [comment]
There is no built in pdf support in justpy but other python libraries might be able to do this. If you found a solution for this please share it. I am moving this to discussions
